tweener/tweener.lua

```diff
@@ -38,28 +37,35 @@ function M.tween(easing_function, from, to, time, callback, update_delta_time)
 
 	local time_elapsed = 0
 	local latest_time = socket.gettime()
+	local timer_table = { id = nil, is_paused = false }
```

Comment:
id is pretty abstract, can we use our `timer_id` still? Even it's a internal code